src/sentry/models/commitauthor.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+
+from sentry.db.models import BoundedPositiveIntegerField, Model, sane_repr
+
+
+class CommitAuthor(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    name = models.CharField(max_length=128, null=True)
+    email = models.EmailField()
```

Comment:
Same here, this applies a strict validation on what an email is, when git is likely more loose. So not sure if these affect us, but does it hurt to just accept the input here as raw text? It makes no difference to the database ultimately.